openlane_update_tool/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Update OpenLane Tool
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
```

Comment:
couldn't this just assume PDK_ROOT? assuming `cache_pdks` is always present as a siblings action in the workflow?

Reply:
See above for required input. Also making it independent of other actions might be useful in the future.